src/etc/inc/util.inc

```diff
@@ -3427,7 +3427,12 @@ function parse_cisco_acl_rule($rule, $devname, $dir, $proto) {
 		$index++;
 		if ((($proto == 'inet') && (is_ipaddrv4(trim($rule[$index])) || (trim($rule[$index]) == "{clientip}"))) ||
 		    (($proto == 'inet6') && (is_ipaddrv6(trim($rule[$index])) || (trim($rule[$index]) == "{clientipv6}")))) {
-			$tmprule .= "from {$rule[$index]} ";
+			if($GLOBALS['attributes']['framed_ip']) {
+				$framed_ip = $GLOBALS['attributes']['framed_ip'];
```

Comment:
Is `$framed_ip` used somewhere else? If not, no need to create it, just use `$GLOBALS['attributes']['framed_ip']` directly in the string below without making the single use variable.